google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/it/BuiltinMetricsIT.java

```diff
@@ -0,0 +1,136 @@
+
+    MetricServiceSettings settings =
+        MetricServiceSettings.newBuilder()
+            .setEndpoint("staging-monitoring.sandbox.googleapis.com:443")
+            .build();
+    // Create a cloud monitoring client
+    metricClient = MetricServiceClient.create(settings);
+  }
+
+  @Test
+  public void testBuiltinMetrics() throws Exception {
+    // Send a MutateRow and ReadRows request
+    testEnvRule
+        .env()
+        .getDataClient()
+        .mutateRow(
+            RowMutation.create(testEnvRule.env().getTableId(), "a-new-key")
+                .setCell(testEnvRule.env().getFamilyId(), "q", "abc"));
+    ArrayList<Row> rows =
+        Lists.newArrayList(
+            testEnvRule
+                .env()
+                .getDataClient()
+                .readRows(Query.create(testEnvRule.env().getTableId()).limit(10)));
+
+    // Sleep 5 minutes so the metrics could be published and precomputation is done
+    Thread.sleep(60 * 5 * 1000);
+
+    ProjectName name = ProjectName.of(testEnvRule.env().getProjectId());
+
+    // Restrict time to last 10 minutes
+    long startMillis = System.currentTimeMillis() - ((60 * 10) * 1000);
+    TimeInterval interval =
+        TimeInterval.newBuilder()
+            .setStartTime(Timestamps.fromMillis(startMillis))
+            .setEndTime(Timestamps.fromMillis(System.currentTimeMillis()))
+            .build();
+
+    for (String view : VIEWS) {
+      // Filter on instance and method name
+      // Verify that metrics are published for MutateRow request
+      String metricFilter =
+          String.format(
+              "metric.type=\"bigtable.googleapis.com/client/%s\" "
+                  + "AND resource.labels.instance=\"%s\" AND metric.labels.method=\"Bigtable.MutateRow\"",
+              view, testEnvRule.env().getInstanceId());
+      ListTimeSeriesRequest.Builder requestBuilder =
+          ListTimeSeriesRequest.newBuilder()
+              .setName(name.toString())
+              .setFilter(metricFilter)
+              .setInterval(interval)
+              .setView(ListTimeSeriesRequest.TimeSeriesView.FULL);
+      ListTimeSeriesResponse response =
+          metricClient.listTimeSeriesCallable().call(requestBuilder.build());
+      Truth.assertThat(response.getTimeSeriesCount()).isGreaterThan(0);
+
+      // Verify that metrics are published for ReadRows request
+      metricFilter =
+          String.format(
+              "metric.type=\"bigtable.googleapis.com/client/operation_latencies\" "
+                  + "AND resource.labels.instance=\"%s\" AND metric.labels.method=\"Bigtable.ReadRows\"",
+              testEnvRule.env().getInstanceId());
+      requestBuilder.setFilter(metricFilter);
+      response = metricClient.listTimeSeriesCallable().call(requestBuilder.build());
+      Truth.assertThat(response.getTimeSeriesCount()).isGreaterThan(0);
+    }
+  }
+
+  @AfterClass
+  public static void tearDown() {
+    metricClient.close();
+  }
```

Comment:
nit move this after setup to make it easier to see that things are getting cleaned up